fluidBuild.config.cjs

```diff
@@ -204,6 +211,12 @@ module.exports = {
 			],
 			gitignore: ["input", "output"],
 		},
+		// eslint-config-fluid specific declarative task to print configs
+		"tsx scripts/print-configs.ts printed-configs": {
```

Comment:
Nit: I can't remember if this particular piece of fluidBuild can be defined in package.json? If we can, this one feels like a good candidate; having it in the same file where the script is would be nice for locality/maintainability.

Reply:
It can't unfortunately.